equipmanager.lic

```diff
@@ -346,9 +346,26 @@ class EquipmentManager
   end
 
   def unload_weapon(name)
-    result = bput("unload my #{name}", 'You unload', 'Your .* fall.* from', 'As you release the string', 'You .* unloading')
+    # Phrases to match:
+    #   (hidden) You remain concealed by your surroundings, convinced that your unloading of the <weapon> went unobserved.
+    #   (hidden) You unload your <weapon> while blended in with your surroundings, but cannot shake the feeling that you drew attention to yourself.
+    #   (one hand empty) You unload the <weapon>.
+    #   (hands full) Your <ammo> falls from your <weapon> to your feet.
+    case bput("unload my #{name}", 'You unload', 'Your .* fall.*', 'As you release the string', 'You .* unloading')
```

Comment:
What does 'As you release the string' mean, in terms of ammo ending up somewhere?

Reply:
I honestly don't know -- it was an original match string so I left it in there. None of my logs over the past year include that phrase.